### ISSUE TITLE
Comments in a BYTESTRING line throw a TypeError

### ISSUE THREAD

**Issue by Kylearan** [plyara/plyara#11]
If a BYTESTRING contains a comment, a ```TypeError: Illegal bytestring character``` is thrown.
Example rule that causes this:
```
rule test {
    strings:
    $a = {
        00 01   // this comment breaks plyara
    }
    condition:
    $a
}
```

**Comment by rshipp** [comment]
Thanks for reporting! This is supposed to be supported, I'll look into it.

**Comment by rshipp** [comment]
I've pushed a new release that should fix this ([v1.2.1](https://github.com/plyara/plyara/releases/tag/v1.2.1)). Let me know if you have any more issues.
